seshdash/generateData.py

```diff
@@ -0,0 +1,50 @@
+from django.utils import timezone
```

Comment:
This file looks like a duplicate of the generate data file in /seshdash/tests/
Can you merge the files so that there is only one module that has all the data generation functions we need? I would put it under. /seshdash/tests/ as data generation is related to testing